cli_tools/daisy/main.go

```diff
@@ -282,3 +285,33 @@ func main() {
 		}
 	}
 }
+
+func printPerfProfile(workflow *daisy.Workflow) {
+	if !*printPerf {
+		return
+	}
+
+	timeRecords := workflow.GetStepTimeRecords()
+	if len(timeRecords) == 0 {
+		return
+	}
+
+	wfStartTime := time.Now()
+	wfEndTime, _ := time.Parse(time.RFC3339, "0001-01-01T00:00:00+00:00")
+	fmt.Println("\nPerf Profile:")
+	for _, r := range timeRecords {
+		if wfStartTime.After(r.StartTime) {
```

Comment:
How can start time of a record be before workflow started?

Reply:
This trying to find the minimum step start time, and use it as workflow start time.